docs/source/content/supervised/logistic_regression.rst

```diff
@@ -5,52 +5,132 @@ Logistic Regression
+Logistic regression works using a linear combination of inputs, so multiple
+information sources can govern the output of the model.
+The parameters of the model are the weights of the various features, and
+represent their relative importance to the result.
+In the equation that follows, you should recognize the formula used in linear regression.
+Logistic regression is, at its base, a transformation from a linear predictor
+to a probability between 0 and 1.
 
-.. figure:: _img/Logistic_Eq.svg
+.. figure:: _img/WikiLogisticEQ.svg
    :alt: Equation
 Ref: https://en.wikipedia.org/wiki/Logistic_regression
 
-The beta values are the weights and x values are the variable inputs.  This formula gives the probability that the input belongs to Class B, which is the output of the function.
+As in linear regression, the beta values are the weights and x values are the variable inputs.
+This formula gives the probability that the input belongs to Class B, which
+is the goal of the logistic regression model.
+
+
+Multinomial logistic regression
+-----------------
+
+Until now, we've been discussing the situation where there are exactly
+two distinct outputs, for example a pass or a fail.
+But, what if there were more than two possible outputs?
+What about the number classification example, where the output can be any digit from 0 to 9?
+
+Well, there is a way to handle that with logistic regression.
+When using the scikit-learn library, as the example code does, the facility is already there.
+With scikit-learn, you can use the multinomial mode and supply any number of
+classes in the training data.
+You can think of the method as creating multiple models and comparing their
+probabilities, but the exact details_ are beyond the scope of this course.
+
+.. _details: https://en.wikipedia.org/wiki/Multinomial_logistic_regression
+
+Code
+-----------------
+
+Check out the example_ for logistic regression in our repository.
+In the example, scikit-learn and numpy are used to train a simple logistic regression model.
+The model is basic, but extensible.
+With logistic regression, more features could be added to the data set
+seamlessly, simply as a column in the 2D arrays.
+
+The code creates a 2D array representing the training input, in this case it is
+1000 x 1, since there are 1000 samples and 1 feature.
+These inputs are scores out of 1000.
+A training output array is also created, with the classification of 1 for
+pass and 0 for fail, based on a threshold.
+Then, scikit-learn's LogisticRegression_ class is used to fit a logistic
+regression classifier to the data.
+After that, the next step is to test for accuracy with a different data set.
+So, we create another 100 random samples to test against, and predict against them using the model.
+.. _LogisticRegression: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html
```

Comment:
You need a newline before this link for it to work properly

Reply:
Fixed in a7fc1f2